src/js/components/FilterButtons.js

```diff
@@ -62,19 +62,21 @@ class FilterButtons extends React.Component {
 }
 
 FilterButtons.defaultProps = {
-  renderButtonContent: function (title) {return title; },
-  onFilterChange: function () {}
+  inverseStyle: false,
+  onFilterChange: function () {},
+  renderButtonContent: function (title) {return title; }
 };
 
 FilterButtons.propTypes = {
-  // Optional function to generate button text. args: (filter, count)
-  renderButtonContent: React.PropTypes.func,
   filters: React.PropTypes.array,
   // The key in itemList that is being filtered
   filterByKey: React.PropTypes.string,
+  inverseStyle: React.PropTypes.bool,
+  itemList: React.PropTypes.array.isRequired,
   // A function that returns the onClick for a filter button given the filter.
   onFilterChange: React.PropTypes.func,
-  itemList: React.PropTypes.array.isRequired,
+  // Optional function to generate button text. args: (filter, count)
```

Comment:
I know it's not you comment. 

I don't trust the `args: (filter, count)`. Could you just remove this portion of the comment?